x-pack/legacy/plugins/security/public/views/management/edit_role/components/privileges/kibana/space_aware_privilege_section/privilege_space_table.tsx

```diff
@@ -203,7 +205,7 @@ export class PrivilegeSpaceTable extends Component<Props, State> {
           } else {
             return (
               <PrivilegeDisplay
-                explanation={basePrivilege}
+                explanation={showCustomize ? undefined : basePrivilege}
```

Comment:
Without this change, if I gave the user global all and then a custom set of privileges at a space granting access to ML, it'd show "Custom" with a lock even though it's truly just Custom:

![custom-lock](https://user-images.githubusercontent.com/627123/60926694-36b47d80-a25c-11e9-9d85-6892843d0b81.gif)